data_pipeline/gmail_data/get_gmail_data.py

```diff
@@ -22,10 +22,11 @@ def get_current_timestamp():
     return datetime.datetime.utcnow().isoformat()
 
 
-@backoff.on_exception(backoff.expo, TimeoutError, max_tries=10)
```

Comment:
perhaps add the backoff back? (it could also be something you could potentially test)